about.html

```diff
@@ -11,54 +11,59 @@
 
 </head>
 
```

Comment:
Great job here, the layout is simple and effective and it is easy to target with flexbox.